scripts/release/download_ERA5_TEC.py

```diff
@@ -0,0 +1,331 @@
+        "the-email-adress-used-as-login@ucar-website.org",
+        "your-uid:your-api-key",
+    ]
+
+    # account file for pyaps3 < and >= 0.3.0
+    cfg_file = parent_dir / "model.cfg"
+    rc_file = Path("~").home() / ".cdsapirc"
+
+    # for ERA5: ~/.cdsapirc
+    if tropo_model == "ERA5" and rc_file.is_file():
+        pass
+
+    # check account info for the following models
+    elif tropo_model in ["ERA5", "ERAI", "MERRA"]:
+        section = MODEL2ARCHIVE_NAME[tropo_model]
+
+        # Read model.cfg file
+        cfg_file = parent_dir / "model.cfg"
+        cfg = ConfigParser()
+        cfg.read(cfg_file)
+
+        # check all required option values
+        for opt in SECTION_OPTS[section]:
+            val = cfg.get(section, opt)
+            if not val or val in default_values:
+                msg = "PYAPS: No account info found "
+                msg += f"for {tropo_model} in {section} section in file: {cfg_file}"
+                raise ValueError(msg)
+
+    return
+
+
+def get_grib_file_names(
+    slc_files: Sequence[Path], grib_dir: Path, frame_id: int
+) -> tuple[list[Path], Bbox]:
+    """Generate the grib file names for each SLC to download.
+
+    Parameters
+    ----------
+    slc_files : Sequence[Path]
+        list of paths to the slc files
+    grib_dir : Path
+        Path to the grib directory
+    frame_id : int
+        DISP-S1 frame id
+
+    Returns
+    -------
+    tuple[list[Path], Bbox]
+        The list of weather model files to be downloaded
+        and the bounding box of the frame
+    """
+    slc_file_list = group_by_date(slc_files)
+    first_date = next(iter(slc_file_list))
+    acquisition_time = get_zero_doppler_time(slc_file_list[first_date][0])
+    hour = closest_weather_model_hour(acquisition_time)
+
+    date_list = [
+        datetime.datetime.strftime(key[0], "%Y%m%d") for key in slc_file_list.keys()
+    ]
+
+    epsg, bounds = get_frame_bbox(frame_id)
+
+    if epsg != 4326:
+        bounds = transform_bounds(CRS.from_epsg(epsg), CRS.from_epsg(4326), *bounds)
+
+    snwe = (bounds[1], bounds[3], bounds[0], bounds[2])
+    area = snwe2str(snwe)
+
+    # grib file list
+    grib_files = []
+    for d in date_list:
+        if area:
+            grib_file = grib_dir / f"ERA5{area}_{d}_{hour}.grb"
+        else:
+            grib_file = grib_dir / f"ERA5_{d}_{hour}.grb"
+        grib_files.append(grib_file)
+
+    return grib_files, snwe
+
+
+def _add_buffer(snwe: tuple[float, float, float, float]):
+    s, n, w, e = snwe
+
+    min_buffer = 1
+    # lat/lon0/1 --> SNWE
+    S = np.floor(min(s, n) - min_buffer).astype(int)
+    N = np.ceil(max(s, n) + min_buffer).astype(int)
+    W = np.floor(min(w, e) - min_buffer).astype(int)
+    E = np.ceil(max(w, e) + min_buffer).astype(int)
+    return S, N, W, E
+
+
+def snwe2str(snwe: tuple[float, float, float, float]):
+    """Get area extent in string."""
+    if not snwe:
+        return None
```

Comment:
```suggestion
def snwe2str(snwe: tuple[float, float, float, float]) -> str:
    """Get area extent in string."""
```
(if the above suggestion is true, then we can remove the check and claim we always return a string)